bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/internal/service/ReadyServiceImpl.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.internal.service;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.locks.ReentrantReadWriteLock;
+import java.util.function.Consumer;
+
+import org.eclipse.smarthome.core.service.ReadyMarker;
+import org.eclipse.smarthome.core.service.ReadyMarkerFilter;
+import org.eclipse.smarthome.core.service.ReadyService;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Implementation of the {@link ReadyService} interface.
+ *
+ * @author Simon Kaufmann - initial contribution and API.
+ *
+ */
+@Component
+public class ReadyServiceImpl implements ReadyService {
+
+    private static final Logger logger = LoggerFactory.getLogger(ReadyServiceImpl.class);
```

Comment:
I didn't see the need to use a logger in a static context, so could we keep the logger instance non static?